welcome/oke_about.adoc

```diff
@@ -299,8 +299,8 @@ s| Feature s| {oke} s| {product-title} s| Operator name
 | Red Hat OpenShift Pipelines Operator | Not Included | Included | OpenShift Pipelines Operator
 | Embedded Component of IBM Cloud Pak and RHT MW Bundles | Not Included | Included | N/A
 | Red Hat OpenShift GitOps | Not Included | Included | OpenShift GitOps
-| Red Hat CodeReady Workspaces | Not Included | Included | CodeReady Workspaces
-| Red Hat CodeReady Containers | Not Included | Included | N/A
+| {openshift-dev-spaces-productname} | Not Included | Included | {openshift-dev-spaces-productname}
```

Comment:
I don't understand the meaning of this line and the `| Not Included | Included |` part.